packages/playground/cli/src/run-cli.ts

```diff
@@ -593,6 +602,7 @@ export async function runCLI(args: RunCLIArgs): Promise<RunCLIServer> {
 					processIdSpaceLength,
 					followSymlinks,
 					trace,
+					disableCookieStore: !args.internalCookieStore,
```

Comment:
let's use a consistent `internalCookieStore` name everywhere and also add a README entry before merging.

Reply:
I have created a PR for it: https://github.com/WordPress/wordpress-playground/pull/2325